dietpi/func/dietpi-banner

```diff
@@ -263,6 +264,18 @@ $GREEN_LINE"
 		(( ${aENABLED[8]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[8]} $GREEN_SEPARATOR $(df -h --output=avail /mnt/dietpi_userdata | mawk 'NR==2 {print $1}' 2>&1)"
 		# Weather
 		(( ${aENABLED[9]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[9]} $GREEN_SEPARATOR $(curl -sSfLm 3 'https://wttr.in/?format=4' 2>&1)"
+		# Certificate
+		local certinfo='No certificate found'
+		local letsencrypt=/etc/letsencrypt
+		if [[ -d "$letsencrypt" ]];
+		then
+			for i in "$letsencrypt"/live/*/cert.pem
+			do
+				certinfo=$(openssl x509 --enddate --noout --in "$i" | mawk '/notAfter=/{print substr($2,1) "-" substr($1,10) "-" substr($4,1) " @ " substr($3,1)}' 2>&1)
+				break
+			done
+		fi
+		(( ${aENABLED[16]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[16]} $GREEN_SEPARATOR $certinfo"
```

Comment:
Suggested change:
- Do not obtain cert info if the banner option is disabled
- Do not assign a variable which is called only a single time
- But declare `i` as local variable
- Instead of checking whether the parent directory exist, check whether the certificate itself exists
```suggestion
		# Let's Encrypt cert status
		if (( ${aENABLED[16]} == 1 ))
		then
			local i certinfo='No certificate found'
			for i in /etc/letsencrypt/live/*/cert.pem
			do
				[[ -f $i ]] || continue
				certinfo=$(openssl x509 --enddate --noout --in "$i" | mawk '/notAfter=/{print "Valid until " substr($2,1) "-" substr($1,10) "-" substr($4,1) " @ " substr($3,1)}' 2>&1)
				break
			done
			echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[16]} $GREEN_SEPARATOR $certinfo"
		fi
```